src/ophys_etl/modules/suite2p_wrapper/__main__.py

```diff
@@ -19,39 +21,53 @@ def run(self):
         self.logger.name = type(self).__name__
         self.logger.setLevel(self.args.pop('log_level'))
 
+        # explicitly set default Suite2P args
+        default_suite2p_args = copy.deepcopy(suite2p.default_ops())
+        default_suite2p_args.update(self.args)
```

Comment:
is this line taking all of suite2p defaults and overriding with our settings? 
The naming of `default_suite2p_args` is a little confusing since we are overriding them. 
Maybe this would be clearer to avoid this confusion
```
default_suite2p_args = copy.deepcopy(suite2p.default_ops())
self.args = {
  **default_suite2p_args,
  **self.args
}
```